services/dynamic-sidecar/tests/unit/test_service_liveness.py

```diff
@@ -0,0 +1,49 @@
+from datetime import timedelta
+
+import pytest
+from simcore_service_dynamic_sidecar.modules.service_liveness import (
+    CouldNotReachServiceError,
+    wait_for_service_liveness,
+)
+
+
+@pytest.mark.parametrize("check_interval", [timedelta(seconds=0.1)])
```

Comment:
why did you add check_interval and timeout as parameters instead of constant values in the test?